src/app/component/eco-news/news-list/news-list-gallery-view/news-list-gallery-view.component.css

```diff
@@ -56,7 +56,7 @@ div {
   flex-wrap: wrap;
```

Comment:
After opening 'eco-news' dashboard, scroll to the screen bottom and after that scroll back to the screen top
![image](https://user-images.githubusercontent.com/54113846/79489761-7480d680-8024-11ea-9aa3-24e42f05fdf9.png)

Reply:
Please fix it

Reply:
Fixed it. Put in wrong class break avoid